src/EventStore.PositionRepository.Gprc/EventStore.PositionRepository.Gprc.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Product>EventStore PositionRepository Grpc</Product>
+    <Authors>Riccardo Di Nuzzo</Authors>
+    <Copyright>Riccardo Di Nuzzo</Copyright>
+    <Description>A repository to save a position and get the current position in EventStore with a GRPC connection</Description>
+    <PackageProjectUrl>https://github.com/riccardone/EventStore.PositionRepository</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/riccardone/EventStore.PositionRepository</RepositoryUrl>
+    <RepositoryType>Github</RepositoryType>
+    <PackageTags>EventSourcing, EventStore, Read Models</PackageTags>
+    <PackageReleaseNotes>New GRPC PositionRepository</PackageReleaseNotes>
+    <PackageId>EventStore.Tools.PositionRepository.Gprc</PackageId>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>1.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="EventStore.Client.Grpc.Streams" Version="23.1.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
could you try to remove the dependency with Newtonsoft and instead use System.Text.Json?